nucypher/cli/commands/stake.py

```diff
@@ -1349,3 +1351,48 @@ def mint(general_config: GroupGeneralConfig,
                           emitter=emitter,
                           chain_name=blockchain.client.chain_name,
                           transaction_type='mint')
+
+
+@stake.command()
+@group_transacting_staker_options
+@option_config_file
+@option_force
+@group_general_config
+@click.option('--external-staker-address',
+              help="The staking address of a staker to manually migrate", type=EIP55_CHECKSUM_ADDRESS)
```

Comment:
What's the rationale for this argument?

Reply:
Will be helpful for pool or if when will remain a few not migrated stakers - we can do this manually and then remove migration code from contract

Reply:
How about something like `--staker-to-migrate`...? We should also clarify that if not specified, the default is the staking address used for transacting.

Reply:
Meh, Comes off a bit redundant to me: `nucypher stake migrate --staker-to-migrate`.